server/routes.js

```diff
@@ -155,5 +155,16 @@ module.exports = (server, app) => {
     );
   });
 
+  server.get('/:collectiveSlug/terms', (req, res) => {
+    const params = { ...req.params, ...req.query };
+    if (req.query.redirect) {
```

Comment:
The goal is to load the redirect URL from the host, not from the query params

Reply:
Hi @Betree , I don't think I know how to do this. How exactly would we get the host data from routes? Could you please point me in the direction?

Reply:
I never did that so I don't have precise steps. But I'd say that you should get the graphql client (maybe from https://github.com/opencollective/opencollective-frontend/blob/60c8bc1a1c0836809c709fdc7d1d315431da04cc/lib/initClient.js#L92 ?) and use it to query the host's settings.

Reply:
I appreciate it, let me check it out.

Reply:
Actually, I don't think we have ES6 imports for that. Or am I missing something?

Reply:
`import` is indeed not supported in that part of the code, you need to use `require`. Babel was disabled here for performances reason: https://github.com/opencollective/opencollective-frontend/pull/2754